chromium_src/chrome/browser/ui/browser.h

```diff
@@ -6,6 +6,7 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_BROWSER_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_BROWSER_H_
 
+#include "brave/browser/ui/brave_tab_strip_model_delegate.h"
```

Comment:
Just double checking this is needed here instead of in the `.cc` file.

Reply:
Oops, you're right Thanks 😄